### ISSUE TITLE
Travis no longer detecting new Pull Requests and commits

### ISSUE THREAD

**Issue by bravo-kernel** [travis-ci/travis-ci#7643]
For about two weeks now new Pull Requests to our repo are not being detected/not triggering Travis and we are pretty much out of ideas (after re-adding the hook, closing/opening PRs, updating `.travis.yml`).

- **Last PR detected:** https://travis-ci.org/FriendsOfCake/crud/pull_requests
- **Example of non-detected new PR:** https://github.com/FriendsOfCake/crud/pull/527

Assistance/pointers highly appreciated.

**Comment by jippi** [comment]
Additionally no requests shows up at https://travis-ci.org/FriendsOfCake/crud/requests when a PR or otherwise is created

We've used the travis-ci UI to enable integration, and tried multiple times to disable/enable it, with different accounts.

Currently the FriendsOfCake/crud travis integration are as follow

![image](https://cloud.githubusercontent.com/assets/22841/25303836/1e381dd0-275b-11e7-845c-74bfcc6f10e5.png)

**Comment by BanzaiMan** [comment]
Could you check your organization's billing status with GitHub? There were similar reports in #7602.

**Comment by jippi** [comment]
@BanzaiMan we are `Free plan, unlimited public repositories`, and have been so since `2016-01-26`

**Comment by BanzaiMan** [comment]
The indications are such that GitHub is not sending notifications for some reason. Rather than going through a third party (Travis CI), could you contact GitHub support to figure out why?

**Comment by bravo-kernel** [comment]
@BanzaiMan contact with GitHub has been made, we will follow up to keep things actual.

**Comment by kaikreuzer** [comment]
In case it helps: I am experiencing the very same since April 10 on https://github.com/openhab/openhab2-addons. But besides Travis, no other webhooks seem to be working either - also mentionbot as well as the Cloudbees PR builder hook stopped working at the very same moment. So I rather suspect an issue on Github side, not at Travis.

**Comment by jippi** [comment]
We've reached out to GH, which seem to have blocked us due to a failed payment ~1 year ago... can't think of another cause that a recent GH change caused this failure.. GH told me to remove all private repos (we had none) and e-mail them to get unblocked again

**Comment by jippi** [comment]
Relevant e-mail from GH  - just hit them up on https://github.com/contact and they will resolve it 

![image](https://cloud.githubusercontent.com/assets/22841/25325312/3cece488-28cb-11e7-8b8c-cdb271a60e2c.png)

**Comment by kaikreuzer** [comment]
Thanks for the info, @jippi. It was indeed the same situation for me and Github support resolved it this morning!

**Comment by kaikreuzer** [comment]
@bravo-kernel I guess the issue can be closed then?

**Comment by BanzaiMan** [comment]
Thanks for the confirmation, everyone.

**Comment by bravo-kernel** [comment]
And thank you mr. @banzaiman for keeping the ticket open until resolved. Hopefully this will help others as well.
